hkm/static/hkm/css/main.css

```diff
@@ -5890,102 +5875,174 @@ a.btn-delete-row {
     background-color: #fff;
     display: inline-block; }
 
-.basket {
+.basket-row {
+  position: relative;
+  padding: 20px;
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: 30% 50% 10% 10%;
```

Comment:
I would consider using `fr` instead of percentage units.

Reply:
Changed percentages to `fr`